### ISSUE TITLE
Joyride Tour guide doesn't explain autoStart:true

### ISSUE THREAD

**Issue by ddgromit** [zurb/joyride#138]
Following the zurb.com [guide steps](http://zurb.com/playground/jquery-joyride-feature-tour-plugin) that this repo references leads to the plugin not starting, because it doesn't mention any kind of `start` method or using the `autoStart:true`.  

It would be very helpful for first-timers trying to integrate this plugin if either that guide was updated to include `autoStart:true`, or the readme informed the user about this requirement.

**Comment by Knight-Yoshi** [comment]
Using Foundation 5, something I found is that you can use something like
$(document).foundation({
topbar: {...},
joyride: 'start'
})
Though I'm not sure how to pass the object of options to it. So that may have to be defined in the data-options

**Comment by Knight-Yoshi** [comment]
Using Foundation 5, something I found is that you can use something like
```
$(document).foundation({
    topbar: {...},
    joyride: 'start',
    ...
})
```
Though I'm not sure how to pass the object of options to it. So that may have to be defined in the data-options

**Comment by btbjosh** [comment]
+1 Doc instruction update needed.

**Comment by bencappello** [comment]
Yes clearly they don't care about this glaring hole in the documentation. I wonder if there is another way to start the tour without autostart

**Comment by filtoid** [comment]
I don't know if this is the "right" way of doing things but if you do the following it might work (mine is at least).

On window load:
$(window).load(function() {
  $("#joyRideTipContent").joyride({
    autoStart : false
  });
});

Then when you want to kick off the joyride you simply call it like this:

$("#my_button").click( function(){
     $("#joyRideTipContent").joyride({
      modal:true,
      expose: true
  });
});

Hope that helps.
